packages/terra-table/src/Table.jsx

```diff
@@ -24,6 +24,11 @@ import getFocusableElements from './utils/focusManagement';
 
 const cx = classNames.bind(styles);
 
+export const rowSelectionModes = {
+  single: 'single',
+  multiple: 'multiple',
```

Comment:
Nit: These keys can be all uppercase

Reply:
Wouldn't this require the props passed in to also be all uppercase?  Is that what we would want?